### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
In almost every issue the first comment is "please fill out the issue template detailing your version".

we'll now also use seperate issue templates for issues and enhancements - hopefully this will turn out positive as well.

From now on, issues (not feature requests) that do not contain this section will be closed.

Also some rewording and hints that we expect copy/pasted text, not screenshots of logs.

### CODE REVIEW

**Comment by hroff-1902** [comment]
"this section" is not applicable for pure enhancement suggestions

would could say that people must mark it with the label "Enhancement" in this case and can then skip filling the section.

**Comment by xmatthias** [comment]
Well we CAN split issue-templates into two ... 

It seems to also support automatic labels - so we can add "triage needed" immediately (we'll need to see if/how that works)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Triage Needed
+assignees: ''
+
+---
+<!-- 
+Have you search for this issue before posting it?
```
Comment (hroff-1902) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
Have you searched for similar issues before posting it?
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+-->
+
+## Describe your environment
+
+  * Operating system: ____
+  * Python Version: _____ (`python -V`)
+  * CCXT version: _____ (`pip freeze | grep ccxt`)
+  * Version: ____ (`freqtrade -V` | `docker-compose run --rm freqtrade -V`)
+  
+Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out..
```
Comment (hroff-1902) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.
```

Comment (hroff-1902) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  3. _____
+  
+### Observed Results:
+
+  * What happened?
+  * What did you expect to happen?
+
+### Relevant code exceptions or logs
+
+Note: Please copy/paste code, no screenshots of logs please.
```
Comment (hroff-1902) [inline .github/ISSUE_TEMPLATE/bug_report.md:44]:
```suggestion
Note: Please copy/paste text of the messages, no screenshots of logs please.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Have you search for this feature before requesting it? It's highly likely that a similar request was already filed.
+-->
+
+## Describe your environment
+(if applicable)
+
+  * Operating system: ____
+  * Python Version: _____ (`python -V`)
+  * CCXT version: _____ (`pip freeze | grep ccxt`)
+  * Version: ____ (`freqtrade -V` | `docker-compose run --rm freqtrade -V`)
```
Comment (hroff-1902) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]:
```suggestion
  * Freqtrade Version: ____ (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)
```

(the pipe `|` can be recognized as the part of the command line, combining both these commands)
same in other places

Comment (xmatthias) [reply .github/ISSUE_TEMPLATE/feature_request.md:21]: so can the or ...

Comment (hroff-1902) [reply .github/ISSUE_TEMPLATE/feature_request.md:21]: yes, but you know that pipes are parts of shell commands, one could simply copy-paste the whole "command" inside the parenthesis...

**Pull request merged**